cmd/kubeadm/app/master/manifests.go

```diff
@@ -353,10 +358,10 @@ func getControllerManagerCommand(cfg *kubeadmapi.MasterConfiguration) []string {
 		"--leader-elect",
 		"--master=127.0.0.1:8080",
 		"--cluster-name="+DefaultClusterName,
-		"--root-ca-file="+kubeadmapi.GlobalEnvParams.HostPKIPath+"/ca.pem",
-		"--service-account-private-key-file="+kubeadmapi.GlobalEnvParams.HostPKIPath+"/apiserver-key.pem",
-		"--cluster-signing-cert-file="+kubeadmapi.GlobalEnvParams.HostPKIPath+"/ca.pem",
-		"--cluster-signing-key-file="+kubeadmapi.GlobalEnvParams.HostPKIPath+"/ca-key.pem",
+		"--root-ca-file="+getCertName(certconstants.CACertName),
+		"--service-account-private-key-file="+getCertName(certconstants.APIServerKeyName),
+		"--cluster-signing-cert-file="+getCertName(certconstants.CACertName),
+		"--cluster-signing-key-file="+getCertName(certconstants.CAKeyName),
 		"--insecure-experimental-approve-all-kubelet-csrs-for-group=system:kubelet-bootstrap",
 	)
 
```

Comment:
In my next PR, I'm refactoring the cloudprovider phase (step 3), and I'm gonna look into if it makes sense to use templates then, it might or might not do.